Packages/Models/Sources/Models/Alias/ServerDate.swift

```diff
@@ -13,7 +13,7 @@ extension ServerDate {
 
   private static var createdAtRelativeFormatter: RelativeDateTimeFormatter = {
     let dateFormatter = RelativeDateTimeFormatter()
-    dateFormatter.unitsStyle = .abbreviated
```

Comment:
Are you sure about that?

Reply:
Yes, tested in French and English. English is same a before, and French went from "-1h." to "il y a 1h".